ExomeDepth/run_ExomeDepth.py

```diff
@@ -185,12 +185,12 @@ def multiprocess_call(multiprocess_list):
         )
 
     action = (
-        "python {csv2vcf} {inputcsv} {refset} {model} {gender} "
+        "python {csv2vcf} {inputcsv} {refset} {calling_model} {gender} "
         " {sampleid} {template} {runid} ").format(
             csv2vcf=settings.csv2vcf,
             inputcsv=inputcsv,
             refset=args.refset,
-            model=multiprocess_list[0],
+            calling_model=multiprocess_list[7],
```

Comment:
you change model to calling_model here, but you never reference it in this script, but still use "model"
https://github.com/UMCUGenetics/Dx_resources/blob/aff23bfceb0d5a606801f60e55dee21753229d08/ExomeDepth/run_ExomeDepth.py#L222
until line 250.

where is calling_model used then?

Reply:
Yes they are. This specific change was made to include the full naming of the high-confident model (i.e. HC_SSv7-2021-1) into the VCF format field.
The other 'model' variables (HC/UMCU) are not changed  as these will impact file naming too much.